### ISSUE TITLE
Is there any plan to refactor the code to python 3?

### ISSUE THREAD

**Issue by zhenlingcn** [DEAP/deap#506]
When I modify the source code, I find the source code is still based on python 2. This leads to an inconvenience for the python 3 developers. I hope we can refactor the source code to python 3, and I believe that this movement will make more developers willing to modify the source code.

**Comment by yxliang01** [comment]
I think it's time to move to Python 3.

**Comment by anton-petrov** [comment]
I propose to start the process of porting. Maybe it's time to new branch? Or someone make new fork, guys?

**Comment by fmder** [comment]
Yes I'm working on porting to Python 3. It is part of the DEAP 2.0 roadmap.

**Comment by fmder** [comment]
The iterable algorithm is a good branch showing the new iterable features and part of moving to python 3 style.

**Comment by gminorcoles** [comment]
hi I am an old deap user and probably returning. will it run on python 3? leaving aside the use of v 3.x features.

**Comment by gminorcoles** [comment]
> hi I am an old deap user and probably returning. will it run on python 3? leaving aside the use of v 3.x features.

well I cloned, installed, and ran spambase.py, and python 3.7 is the only python on my machine, and it at least runs, so maybe not such a big deal

**Comment by fmder** [comment]
Yes 2to3 will take care of it
